arangod/Replication2/Storage/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+target_sources(arango_replication2 PRIVATE
+  ILogPersistor.h
+  IStatePersistor.h
+  IStorageEngineMethods.h
+  PersistedStateInfo.h
+  )
+
+add_subdirectory(RocksDB)
```

Comment:
```suggestion
add_subdirectory(RocksDB)

```

Reply:
I do not think this is important, GitHub did just add this annoying red sign to the line ;)